libethcore/SealEngine.cpp

```diff
@@ -151,3 +158,80 @@ u256 SealEngineBase::blockReward(u256 const& _blockNumber) const
     EVMSchedule const& schedule{evmSchedule(_blockNumber)};
     return chainParams().blockReward(schedule);
 }
+
+namespace dev
```

Comment:
Better put everything in the file into these and remove `using namespace dev` `using namespace eth`